ci-operator/config/openshift/os/openshift-os-master.yaml

```diff
@@ -53,6 +64,52 @@ tests:
   container:
     from: build-test-qemu-img
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: periodic-os-build
+  cluster: build02
+  cron: '@daily'
+  steps:
+    test:
+    - as: oc-login
+      cli: latest
+      commands: |
+        #!/bin/bash
+        set -euo pipefail
+        # To inject the oc CLI, ci-operator builds a new container from the
+        # provided container dependency (from:) and pushes it to the namespace
+        # Imagestream before creating a new pod to run the commands in. Because
+        # of the size of the coreos-assembler image, this adds quite some time
+        # to the process. So lets use a much smaller Fedora base image.
+        oc login https://api.ci.l2s4.p1.openshiftapps.com:6443 --token="$(cat /service-account-token/image-pusher-service-account-token)"
+        oc registry login --registry=registry.ci.openshift.org --to="$SHARED_DIR/dockercfg.json"
+      credentials:
+      - mount_path: /service-account-token
+        name: rhcos-devel-image-pusher-service-account-token
+        namespace: test-credentials
+      from: fedora
+      resources:
+        limits:
+          devices.kubevirt.io/kvm: "1"
+        requests:
+          cpu: 1000m
+          devices.kubevirt.io/kvm: "1"
+          memory: 3Gi
+    - as: build-test-and-push-img
+      commands: |
+        #!/bin/bash
+        set -euo pipefail
+
+        /src/ci/build-test-qemu.sh
+
+        export REGISTRY_AUTH_FILE="$SHARED_DIR/dockercfg.json"
+        cosa push-container registry.ci.openshift.org/rhcos-devel/machine-os-oci-content-cloud:latest
```

Comment:
Did we already add an imagestream object for this?  I see it already on the server, but that may be because of rehearsals?

Can we bikeshed the naming a bit?  At the current time, `-oci` is important but I feel in the future it will just be a normal thing.  I also think a big change in this effort is that users/customers the image will be a "touch point" - less an implementation detail.  So the fact that it is RHEL (and CoreOS) is more important.

Perhaps e.g. `rhel-coreos:latest` ?  Nice and short and direct?

Versioning is going to be important in the future; we could go ahead and make this `rhel-coreos:4.11` actually right now and not even have `:latest`?

Reply:
+1 rhel-coreos or something short like that 

however just to :bike:  latest is the convention most ocp repos use for current non-release

Reply:
Yes, I did add that Imagestream. We can certainly bikeshed the name right now; I like the name `rhel-coreos` instead. A further question: Do we want to have additional tags such as the build ID, etc.? Or is the OCP version (`4.11`) sufficient for now?

Reply:
I think 1 piece of distinguishing info would probably be good like a build-id tag?

Reply:
> I think 1 piece of distinguishing info would probably be good like a build-id tag?

Wholly agreed. I think the place to add that is inside the `cosa push-container` command since it has that information at the ready. To disambiguate: Which `build-id` you mean? Do you mean the OpenShift CI build ID (e.g., `1496976030290677760`) or the `cosa` build ID (e.g., `411.84.202202281627-0-x86_64`)? Or maybe both?

Reply:
I've done the following:

- Opened https://github.com/coreos/coreos-assembler/pull/2734 to enable `cosa push-container` to tag the container with the build ID and architecture since I think that's the most appropriate place for that logic to live.
- Created the `rhel-coreos` imagestream in the `rhcos-devel` namespace.
- Will modify the job code (this PR) to push to the new Imagestream.

The new push code will push to the following tags:

- `rhel-coreos:<cosa build ID>-<arch>`
- `rhel-coreos:latest`
- `rhel-coreos:4.11`